.github/workflows/pull-request.yml

```diff
@@ -39,6 +39,8 @@ jobs:
         # TODO: remove when lief releases the 3.12 wheel (https://github.com/lief-project/LIEF/issues/990)
         if: matrix.python-version != 3.12
         run: |
+          pip3 install mypy-protobuf protobuf grpcio-tools
```

Comment:
- use `python3 -m pip` to be consistent with the rest of the code snippets
- that way, if one has several python installations, the snippets are still guaranteed to work